test/t8n/t8n.cpp

```diff
@@ -54,6 +55,9 @@ int main(int argc, const char* argv[])
             output_result_file = argv[i];
         else if (arg == "--output.alloc" && ++i < argc)
             output_alloc_file = argv[i];
+        else if (arg == "--state.reward" && ++i < argc)
+            if (std::string_view(argv[i]) != "-1")
```

Comment:
You have to combine all 3 conditions. You can be fancy by doing `argv[i] != "-1"sv`.

Reply:
error: no matching literal operator for call to 'operator""sv' with arguments of types 'const char *' and 'unsigned long', and no matching literal operator template
        else if (arg == "--state.reward" && ++i < argc && argv[i] != "-1"sv)

Reply:
`using namespace std::literals`.